code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -1034,7 +1034,7 @@ pub mod pallet {
 			Self::deposit_event(Event::<T>::SplitPosition {
 				positions: vec![
 					(*fnft_collection_id, *fnft_instance_id, old_position_stake),
-					(*fnft_collection_id, *fnft_instance_id, new_stake.stake),
+					(*fnft_collection_id, new_fnft_instance_id, new_stake.stake),
```

Comment:
Are there no tests failing for this? We need to make sure that there is a test assertion for this

Reply:
That test is currently ignored, apparently while something is being solved.

Reply:
I see, I think @PoisonPhang is planning to fix that soon. Let's merge this then.